frontend/src/machines/paymentMachine.ts

```diff
@@ -1,8 +1,130 @@
-import { createMachine } from "xstate";
+import { assign, createMachine } from "xstate";
+import { Member } from "../__generated__/api";
+import { match } from "ts-pattern";
 
-const paymentMachine = createMachine({
+interface Context {
+  memberDetail: Member[];
+  memberDetailError: string | null;
+  payments: [];
+  paymentsError: string | null;
+  submitPayment: [];
```

Comment:
`submitPayment: []` payment ini buat nampung apa ya ? kayaknya disini gak perlu nyimpen response BE dari submit payment ?

Reply:
oiya yaa, okeey tak hapus kalau gitu